features/url-rewriters/cloudinary.md

```diff
@@ -37,5 +37,6 @@ The following Cloudinary transformations are supported for translation:
 | <p>start offset</p> | <ul> <li>`so_10.5599` => `so-10.56`</li> </ul> |  |
 | <p>end offset</p> | <ul> <li>`eo_10.5522` => `eo-10.55`</li> </ul> |  |
 | <p>duration</p> | <ul> <li>`du_10.5500` => `du-10.55`</li> </ul> |  |
+| <p>zoom</p> | <ul> <li>`z_2.0` => `z-2.0`</li><li>`z_0.5` => `z-0.5`</li></ul> | Final output from ImageKit may vary from output from Cloudinary. |
```

Comment:
remove the `Final output from ImageKit may vary from output from Cloudinary.`. We have used that string for the effects row in the table because the output sometimes differs wildly there

Reply:
done